### ISSUE TITLE
Sync Text Scroll Position with Audio

### ISSUE THREAD

**Issue by caudaganesh** [sonnylazuardi/alkitab-app#14]
I think it will be great if we can sync the audio with the scroll position of the text

**Comment by sonnylazuardi** [comment]
agree. but how can we get the marker position for each verse? or it should scroll based on percentage?

**Comment by caudaganesh** [comment]
I've been experimenting with some methods, basically we can scroll based on percentage of the audio file duration, not 100% accurate but mostly okay. We also need to adapt with some parts especially some parts where has music intro before the actual verse begin.

But still looking for better solution, maybe we can use audio to text, and sync them based on that data, never tried this method but it seems doable.

I think we need further investigation for this feature.
